projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -1187,6 +1190,42 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy {
         return this._visibleWhenCollapsed;
     }
 
+    private _formatOptions: IColumnPipeArgs = { format: DEFAULT_DATE_FORMAT, digitsInfo: DEFAULT_DIGITS_INFO };
+    /**
+     * @remarks
+     * Provide parameters for DatePipe and DecimalPipe to customize the display format for date and numeric columns.
+     * Accepts an `IColumnPipeArgs` object with any of the `format`, `timezone` and `digitsInfo` properties.
+     * For more details see https://angular.io/api/common/DatePipe and https://angular.io/api/common/DecimalPipe
+     * @param IColumnPipeArgs object.
+     *
+     * @memberof IgxColumnComponent
+     */
+    @Input()
+    /**
+     * Sets the formatOptions.
+     * @example
+     * ```typescript
+     * const formatOptions: IColumnPipeArgs = {
+     *      format: 'longDate',
+     *      timezone: 'UTC'
+     * }
+     * ```
+     * ```html
+     * <igx-column dataType="date" formatOptions="formatOptions"></igx-column>
+     * <igx-column dataType="number" formatOptions="{ digitsInfo: '1.1-2' }"></igx-column>
+     * ```
+     */
+    set formatOptions(value: IColumnPipeArgs) {
```

Comment:
Can't we also have the input named `columnPipeArgs`?

Reply:
We can. I will change that.

Reply:
I agree with Hristo, lets name it `pipeArgs`